### ISSUE TITLE
[RFE] Add ability to add environmental variables to LB's via the UI

### ISSUE THREAD

**Issue by aemneina** [rancher/rancher#8373]
**Rancher Versions:**
Server: v1.5.3

**Steps to Reproduce:**
Enhancement request to add environmental variables via the UI for load balancers. understandably we cant have all the things, but this is super helpful especially when working on a development setup that uses vagrant.

**Comment by vincent99** [comment]
What exactly is this useful for?

**Comment by bastiandg** [comment]
This was created after I opened a support ticket.

I'm not able to start the rancher load balancers in a vagrant machine. They constantly restart with

```
time="2017-03-30T10:34:44Z" level=info msg="Starting Rancher LB service"
time="2017-03-30T10:34:44Z" level=info msg="LB controller: rancher"
time="2017-03-30T10:34:44Z" level=info msg="LB provider: haproxy"
time="2017-03-30T10:34:44Z" level=info msg="starting rancher controller"
time="2017-03-30T10:34:44Z" level=info msg="Healthcheck handler is
listening on :10241"
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x63 pc=0x7f841b940259]

runtime stack:
runtime.throw(0x278eca0, 0x2a)
/usr/local/go/src/runtime/panic.go:547 +0x90
runtime.sigpanic()
/usr/local/go/src/runtime/sigpanic_unix.go:12 +0x5a
[...]
```

To mitigate this issue I had to add the label `GODEBUG=netdns=go` to the load balancer (and the healthcheck as well).

So yes @vincent99 as you implied, fixing this isn't prio 1. And no I don't necessarily need a fix.

I'd prefer you to raise the prio on this: https://github.com/rancher/rancher/issues/6111 (open since october) which is the core of this issue.

**Comment by aemneina** [comment]
Looks like this was addressed in 1.6.0 #6111
